oak-run-commons/src/main/java/org/apache/jackrabbit/oak/fixture/DataStoreUtils.java

```diff
@@ -146,26 +150,52 @@ public static void deleteBucket(String bucket, Map<String, ?> map, Date date) th
     }
 
     public static void deleteAzureContainer(Map<String, ?> config, String containerName) throws Exception {
+        if (config == null) {
+            return;
```

Comment:
Add  a warn log here

Reply:
done

Reply:
cannot initialize*

Reply:
thanks for pointing this our, changes done!